indico/modules/legal/__init__.py

```diff
@@ -51,3 +56,44 @@ def _inject_privacy_footer(**kwargs):
     url = legal_settings.get('privacy_policy_url')
     if url or legal_settings.get('privacy_policy'):
         return render_template('legal/privacy_footer.html', url=url)
+
+
+@template_hook('global-announcement', markup=False)
+def _inject_announcement_header(**kwargs):
+    # Don't show if no user or not admin
+    if not session.user or not session.user.is_admin:
+        return
```

Comment:
This _should_ be hidden for admins, right?

Reply:
No, this one is visible for admins only:
* If enabled, normal users will be force redirected to a page to accept the terms before they continue. They never see a global-announcement banner.
* Admins can continue to browse normally, but will receive a global-announcement banner at the top telling them they should accept the terms.

Reply:
I've updated the checks so the banner doesn't appear when accepting terms is not forced, to align with the normal user experience.